### DISCUSSION TITLE
SDXL - 43+ Stable Diffusion Tutorials, Automatic1111 Web UI and Google Colab Guides, NMKD GUI, RunPod, DreamBooth - LoRA & Textual Inversion Training, Model Injection, CivitAI & Hugging Face Custom Models, Txt2Img, Img2Img, Video To Animation, Batch Processing, AI Upscaling

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [JoePenna/Dreambooth-Stable-Diffusion#151] [Show and tell]
[![image](https://img.shields.io/discord/772774097734074388?label=Discord&logo=discord)](https://discord.com/servers/software-engineering-courses-secourses-772774097734074388) [![Hits](https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2FFurkanGozukara%2FStable-Diffusion&count_bg=%2379C83D&title_bg=%239E0F0F&icon=apachespark.svg&icon_color=%23E7E7E7&title=views&edge_flat=false)](https://hits.seeyoufarm.com) [![Twitter Follow Furkan Gözükara](https://img.shields.io/badge/Twitter-Follow%20Me-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/GozukaraFurkan)

[![YouTube Channel](https://img.shields.io/badge/YouTube-SECourses-C50C0C?style=for-the-badge&logo=youtube)](https://www.youtube.com/SECourses) [![Patreon](https://img.shields.io/badge/Patreon-Support%20Me-F2EB0E?style=for-the-badge&logo=patreon)](https://www.patreon.com/SECourses) [![Furkan Gözükara LinkedIn](https://img.shields.io/badge/LinkedIn-Follow%20Me-0077B5?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/furkangozukara/) 

# Expert-Level Tutorials on Stable Diffusion: Master Advanced Techniques and Strategies

Greetings everyone. I am Dr. Furkan Gözükara. I am an Assistant Professor in Software Engineering department  of a private university (have PhD in Computer Engineering). My professional programming skill is unfortunately C# not Python :) 

My linkedin : [**https://www.linkedin.com/in/furkangozukara**](https://www.linkedin.com/in/furkangozukara/)

### Our discord to get more help : [**https://discord.com/servers/software-engineering-courses-secourses-772774097734074388**](https://discord.com/servers/software-engineering-courses-secourses-772774097734074388)

I am keeping this list up-to-date. I got upcoming new awesome video ideas. Trying to find time to do that.

### All videos have manually fixed subtitles and properly prepared video chapters. You can watch with these perfect subtitles or look for the chapters you are interested in.

Since my profession is teaching, I usually do not skip any of the important parts. Therefore, you may find my videos a little bit longer.

Playlist link on YouTube: [**Stable Diffusion Tutorials, Automatic1111 Web UI & Google Colab Guides, DreamBooth, Textual Inversion / Embedding, LoRA, AI Upscaling, Video to Anime**](https://www.youtube.com/watch?v=mnCY8uM7E50&list=PL_pbwdIyffsmclLl0O144nQRnezKlNdx3)

1.) Automatic1111 Web UI - PC - Free

[**How To Install Python, Setup Virtual Environment VENV, Set Default Python System Path & Install Git**](https://youtu.be/B5U7LJOvH6g)

[![image](https://cdn-uploads.huggingface.co/production/uploads/6345bd89fe134dfd7a0dba40/S4mKtVuWlCibihsTHL9TS.png)](https://youtu.be/B5U7LJOvH6g)

2.) Automatic1111 Web UI - PC - Free

[**Easiest Way to Install & Run Stable Diffusion Web UI on PC by Using Open Source Automatic Installer**](https://www.youtube.com/watch?v=AZg6vzWHOTA)

[![image](https://user-images.githubusercontent.com/19240467/218344261-aa236e18-152f-4287-b4fd-fa09c8f57a3f.png)](https://www.youtube.com/watch?v=AZg6vzWHOTA)

3.) Automatic1111 Web UI - PC - Free

[**How to use Stable Diffusion V2.1 and Different Models in the Web UI - SD 1.5 vs 2.1 vs Anything V3**](https://www.youtube.com/watch?v=aAyvsX-EpG4)

[![image](https://user-images.githubusercontent.com/19240467/218344276-af8f2fa2-4fdb-4454-9c8f-bd3ef4d2c92a.png)](https://www.youtube.com/watch?v=aAyvsX-EpG4)

4.) Automatic1111 Web UI - PC - Free

[**Zero To Hero Stable Diffusion DreamBooth Tutorial By Using Automatic1111 Web UI - Ultra Detailed**](https://www.youtube.com/watch?v=Bdl-jWR3Ukc)

[![image](https://user-images.githubusercontent.com/19240467/218344301-04f91cf4-fa35-4975-8c3d-9951c765839a.png)](https://www.youtube.com/watch?v=Bdl-jWR3Ukc)

5.) Automatic1111 Web UI - PC - Free

[**DreamBooth Got Buffed - 22 January Update - Much Better Success Train Stable Diffusion Models Web UI**](https://www.youtube.com/watch?v=KwxNcGhHuLY)

[![image](https://user-images.githubusercontent.com/19240467/218344369-97b68dd7-732d-4ca3-9acc-a87984ebe0f0.png)](https://www.youtube.com/watch?v=KwxNcGhHuLY)

6.) Automatic1111 Web UI - PC - Free

[**How to Inject Your Trained Subject e.g. Your Face Into Any Custom Stable Diffusion Model By Web UI**](https://www.youtube.com/watch?v=s25hcW4zq4M)

[![image](https://user-images.githubusercontent.com/19240467/218344509-01d70965-aeea-4096-bc29-7a005b4d47a6.png)](https://www.youtube.com/watch?v=s25hcW4zq4M)
 
7.) Automatic1111 Web UI - PC - Free

[**How To Do Stable Diffusion LORA Training By Using Web UI On Different Models - Tested SD 1.5, SD 2.1**](https://www.youtube.com/watch?v=mfaqqL5yOO4)

[![image](https://user-images.githubusercontent.com/19240467/218344459-bd4554b0-b57b-4079-aaea-ed93d8be95ed.png)](https://www.youtube.com/watch?v=mfaqqL5yOO4)

8.) Automatic1111 Web UI - PC - Free

[**8 GB LoRA Training - Fix CUDA & xformers For DreamBooth and Textual Inversion in Automatic1111 SD UI**](https://www.youtube.com/watch?v=O01BrQwOd-Q)

[![image](https://user-images.githubusercontent.com/19240467/218344491-52ac51d8-6556-4abc-b2fb-d640a46c48a2.png)](https://www.youtube.com/watch?v=O01BrQwOd-Q)

9.) Automatic1111 Web UI - PC - Free

[**How To Do Stable Diffusion Textual Inversion (TI) / Text Embeddings By Automatic1111 Web UI Tutorial**](https://www.youtube.com/watch?v=dNOpWt-epdQ)

[![image](https://user-images.githubusercontent.com/19240467/218344538-d5f0329d-b0e9-44ed-aaf0-5e4bb134afb7.png)](https://www.youtube.com/watch?v=dNOpWt-epdQ)

10.) Automatic1111 Web UI - PC - Free

[**How To Generate Stunning Epic Text By Stable Diffusion AI - No Photoshop - For Free - Depth-To-Image**](https://www.youtube.com/watch?v=TBq1bhY8BOc)

[![image](https://user-images.githubusercontent.com/19240467/218344579-fda1e9b8-a810-48af-9dcb-f47e87afee9e.png)](https://www.youtube.com/watch?v=TBq1bhY8BOc)

11.) Python Code - Hugging Face Diffusers Script - PC - Free

[**How to Run and Convert Stable Diffusion Diffusers (.bin Weights) & Dreambooth Models to CKPT File**](https://www.youtube.com/watch?v=-6CA18MS0pY)

[![image](https://user-images.githubusercontent.com/19240467/218344677-3f812cf3-db37-4ccb-8f81-99b8a1d5ef00.png)](https://www.youtube.com/watch?v=-6CA18MS0pY)

12.) NMKD Stable Diffusion GUI - Open Source - PC - Free

[**Forget Photoshop - How To Transform Images With Text Prompts using InstructPix2Pix Model in NMKD GUI**](https://www.youtube.com/watch?v=EPRa8EZl9Os)

[![image](https://user-images.githubusercontent.com/19240467/218344868-3232f875-b2c5-4caa-b59b-9d0fd683c06b.png)](https://www.youtube.com/watch?v=EPRa8EZl9Os)

13.) Google Colab Free - Cloud - No PC Is Required

[**Transform Your Selfie into a Stunning AI Avatar with Stable Diffusion - Better than Lensa for Free**](https://www.youtube.com/watch?v=mnCY8uM7E50)

[![image](https://user-images.githubusercontent.com/19240467/218344900-286cded5-0171-4b9e-9354-7adf4bada612.png)](https://www.youtube.com/watch?v=mnCY8uM7E50)

14.) Google Colab Free - Cloud - No PC Is Required

[**Stable Diffusion Google Colab, Continue, Directory, Transfer, Clone, Custom Models, CKPT SafeTensors**](https://www.youtube.com/watch?v=kIyqAdd_i10)

[![image](https://user-images.githubusercontent.com/19240467/218344930-95956805-6a6e-46ee-8885-64043246d79b.png)](https://www.youtube.com/watch?v=kIyqAdd_i10)

15.) Automatic1111 Web UI - PC - Free

[**Become A Stable Diffusion Prompt Master By Using DAAM - Attention Heatmap For Each Used Token - Word**](https://www.youtube.com/watch?v=XiKyEKJrTLQ)

[![image](https://user-images.githubusercontent.com/19240467/218345146-54076e5d-230a-4774-8d6a-8358cbd15f78.png)](https://www.youtube.com/watch?v=XiKyEKJrTLQ)

16.) Python Script - Gradio Based - ControlNet - PC - Free

[**Transform Your Sketches into Masterpieces with Stable Diffusion ControlNet AI - How To Use Tutorial**](https://www.youtube.com/watch?v=YJebdQ30UZQ)

[![image](https://user-images.githubusercontent.com/19240467/218345328-ada437bf-5eb4-478e-a951-84486a42995d.png)](https://www.youtube.com/watch?v=YJebdQ30UZQ)

17.) Automatic1111 Web UI - PC - Free

[**Sketches into Epic Art with 1 Click: A Guide to Stable Diffusion ControlNet in Automatic1111 Web UI**](https://www.youtube.com/watch?v=vhqqmkTBMlU) 

[![image](https://user-images.githubusercontent.com/19240467/218806127-c84d1ff8-d5bb-41b0-bdef-6922568792b9.png)](https://www.youtube.com/watch?v=vhqqmkTBMlU)

18.) RunPod - Automatic1111 Web UI - Cloud - Paid - No PC Is Required

[**Ultimate RunPod Tutorial For Stable Diffusion - Automatic1111 - Data Transfers, Extensions, CivitAI**](https://www.youtube.com/watch?v=QN1vdGhjcRc) 

[![image](https://cdn-uploads.huggingface.co/production/uploads/6345bd89fe134dfd7a0dba40/NLbuo08ixbjt5t3iG5ioG.png)](https://www.youtube.com/watch?v=QN1vdGhjcRc)

19.) RunPod - Automatic1111 Web UI - Cloud - Paid - No PC Is Required

[**How To Install DreamBooth & Automatic1111 On RunPod & Latest Libraries - 2x Speed Up - cudDNN - CUDA**](https://youtu.be/c_S2kFAefTQ)

[![image](https://i.imgur.com/5JbyWKE.png)](https://youtu.be/c_S2kFAefTQ)

20.) Automatic1111 Web UI - PC - Free

[**Fantastic New ControlNet OpenPose Editor Extension & Image Mixing - Stable Diffusion Web UI Tutorial**](https://youtu.be/iFRdrRyAQdQ) 

[![image](https://user-images.githubusercontent.com/19240467/220776337-3abce5a3-bb17-4240-8400-4e633562ecc8.png)](https://youtu.be/iFRdrRyAQdQ)

21.) Automatic1111 Web UI - PC - Free

[**Automatic1111 Stable Diffusion DreamBooth Guide: Optimal Classification Images Count Comparison Test**](https://youtu.be/Tb4IYIYm4os) 

[![image](https://user-images.githubusercontent.com/19240467/221384116-e42d6f37-a068-4a2a-9bda-11ac47f33faa.png)](https://youtu.be/Tb4IYIYm4os)

22.) Automatic1111 Web UI - PC - Free

[**Epic Web UI DreamBooth Update - New Best Settings - 10 Stable Diffusion Training Compared on RunPods**](https://youtu.be/sRdtVanSRl4) 

[![image](https://user-images.githubusercontent.com/19240467/222991604-ceed12bc-0bc9-4f16-82fe-e6779132e00c.png)](https://youtu.be/sRdtVanSRl4)

23.) Automatic1111 Web UI - PC - Free

[**New Style Transfer Extension, ControlNet of Automatic1111 Stable Diffusion T2I-Adapter Color Control**](https://youtu.be/tXaQAkOgezQ)

[![image](https://user-images.githubusercontent.com/19240467/223283277-eaaf6e53-df43-40ac-8096-c08f9a14cc8d.png)](https://youtu.be/tXaQAkOgezQ)

24.) Automatic1111 Web UI - PC - Free

[**Generate Text Arts & Fantastic Logos By Using ControlNet Stable Diffusion Web UI For Free Tutorial**](https://youtu.be/C_mJI4U23nQ)

[![image](https://user-images.githubusercontent.com/19240467/224442765-ba241f71-b412-4f5b-bf39-506e9682e336.png)](https://youtu.be/C_mJI4U23nQ)

25.) Automatic1111 Web UI - PC - Free

[**How To Install New DREAMBOOTH & Torch 2 On Automatic1111 Web UI PC For Epic Performance Gains Guide**](https://youtu.be/pom3nQejaTs)

[![image](https://user-images.githubusercontent.com/19240467/226115542-72db7e7e-cee0-4e3a-82c4-12348e2b237e.png)](https://youtu.be/pom3nQejaTs)

26.) Automatic1111 Web UI - PC - Free

[**Training Midjourney Level Style And Yourself Into The SD 1.5 Model via DreamBooth Stable Diffusion**](https://youtu.be/m-UVVY_syP0)

[![image](https://user-images.githubusercontent.com/19240467/226378438-fe70f09e-94a8-4d1d-9468-e44dca99aac7.png)](https://youtu.be/m-UVVY_syP0)

27.) Automatic1111 Web UI - PC - Free

[**Video To Anime - Generate An EPIC Animation From Your Phone Recording By Using Stable Diffusion AI**](https://youtu.be/kmT-z2lqEPQ)

[![image](https://user-images.githubusercontent.com/19240467/228096548-5f6add70-ca04-4bec-8c33-24d243227532.png)](https://youtu.be/kmT-z2lqEPQ)

28.) Python Script - Jupyter Based - PC - Free

[**Midjourney Level NEW Open Source Kandinsky 2.1 Beats Stable Diffusion - Installation And Usage Guide**](https://youtu.be/dYt9xJ7dnpU)

[![image](https://user-images.githubusercontent.com/19240467/230183162-8a6f7e84-dcd9-45b5-a94c-b93a10778f42.png)](https://youtu.be/dYt9xJ7dnpU)

29.) Automatic1111 Web UI - PC - Free

[**RTX 3090 vs RTX 3060 Ultimate Showdown for Stable Diffusion, ML, AI & Video Rendering Performance**](https://youtu.be/lgP1LNnaUaQ)

[![image](https://user-images.githubusercontent.com/19240467/231303430-63d801cf-3c5a-4c20-b445-bb682febfa4e.png)](https://youtu.be/lgP1LNnaUaQ)

30.) Kohya Web UI - Automatic1111 Web UI - PC - Free

[**Generate Studio Quality Realistic Photos By Kohya LoRA Stable Diffusion Training - Full Tutorial**](https://youtu.be/TpuDOsuKIBo)

[![image](https://user-images.githubusercontent.com/19240467/235155355-83ff14e5-a3c8-4ae8-83a5-6d2573189a22.png)](https://youtu.be/TpuDOsuKIBo)

31.) Kaggle NoteBook - Free

[**DeepFloyd IF By Stability AI - Is It Stable Diffusion XL or Version 3? We Review and Show How To Use**](https://youtu.be/R2fEocf-MU8)

[![image](https://user-images.githubusercontent.com/19240467/235505544-2ba77ef2-3928-4c44-aba8-2536aebbfb60.png)](https://youtu.be/R2fEocf-MU8)

32.) Python Script - Automatic1111 Web UI - PC - Free

[**How To Find Best Stable Diffusion Generated Images By Using DeepFace AI - DreamBooth / LoRA Training**](https://youtu.be/343I11mhnXs)

[![image](https://user-images.githubusercontent.com/19240467/236293388-6254ff84-0866-4bd4-a5d4-2db3c42be3f0.png)](https://youtu.be/343I11mhnXs)

33.) Kohya Web UI - RunPod - Paid

[**How To Install And Use Kohya LoRA GUI / Web UI on RunPod IO With Stable Diffusion & Automatic1111**](https://youtu.be/3uzCNrQao3o)

[![image](https://i.imgur.com/q7FPRIO.png)](https://youtu.be/3uzCNrQao3o)

34.) PC - Google Colab - Free

[**Mind-Blowing Deepfake Tutorial: Turn Anyone into Your Favorite Movie Star! PC & Google Colab - roop**](https://youtu.be/OI1LEN-SgLM)

[![image](https://i.imgur.com/ICWeeV0.png)](https://youtu.be/OI1LEN-SgLM)

35.) Automatic1111 Web UI - PC - Free

[**Stable Diffusion Now Has The Photoshop Generative Fill Feature With ControlNet Extension - Tutorial**](https://youtu.be/ot5GkaxHPzk)

[![image](https://i.imgur.com/Cf4z9J7.png)](https://youtu.be/ot5GkaxHPzk)

36.) Automatic1111 Web UI - PC - Free

[**Human Cropping Script & 4K+ Resolution Class / Reg Images For Stable Diffusion DreamBooth / LoRA**](https://youtu.be/QTYX0tgA5ho)

[![image](https://i.imgur.com/oyZxQTu.png)](https://youtu.be/QTYX0tgA5ho)

37.) Automatic1111 Web UI - PC - Free

[**Stable Diffusion 2 NEW Image Post Processing Scripts And Best Class / Regularization Images Datasets**](https://youtu.be/olX1mySE8HA)

[![image](https://i.imgur.com/cGY0AWk.png)](https://youtu.be/olX1mySE8HA)

38.) Automatic1111 Web UI - PC - Free

[**How To Use Roop DeepFake On RunPod Step By Step Tutorial With Custom Made Auto Installer Script**](https://youtu.be/jD1ZSd9aFHg)

[![image](https://i.imgur.com/Qgyv3Kq.png)](https://youtu.be/jD1ZSd9aFHg)

39.) RunPod - Automatic1111 Web UI - Cloud - Paid - No PC Is Required

[**How To Install DreamBooth & Automatic1111 On RunPod & Latest Libraries - 2x Speed Up - cudDNN - CUDA**](https://youtu.be/c_S2kFAefTQ)

[![image](https://s3.amazonaws.com/moonup/production/uploads/6345bd89fe134dfd7a0dba40/uNesGHfT0G2maEonPl_mq.png)](https://youtu.be/c_S2kFAefTQ)

40.) Automatic1111 Web UI - PC - Free + RunPod

[**Zero to Hero ControlNet Tutorial: Stable Diffusion Web UI Extension | Complete Feature Guide**](https://youtu.be/3E5fhFQUVLo)

[![image](https://s3.amazonaws.com/moonup/production/uploads/6345bd89fe134dfd7a0dba40/OU_mG6YvrSOAZLboBwauu.png)](https://youtu.be/3E5fhFQUVLo)

41.) Automatic1111 Web UI - PC - Free + RunPod

[**The END of Photography - Use AI to Make Your Own Studio Photos, FREE Via DreamBooth Training**](https://youtu.be/g0wXIcRhkJk)

[![image](https://cdn-uploads.huggingface.co/production/uploads/6345bd89fe134dfd7a0dba40/vp0tLPFBzWzNj9G0ev5TV.png)](https://youtu.be/g0wXIcRhkJk)

42.) Google Colab - Gradio - Free

[**How To Use Stable Diffusion XL (SDXL 0.9) On Google Colab For Free**](https://youtu.be/s2MQqmv6yAg)

[![image](https://cdn-uploads.huggingface.co/production/uploads/6345bd89fe134dfd7a0dba40/A2RflDXjgXjg6Z2Neq9gv.png)](https://youtu.be/s2MQqmv6yAg)

43.) Local - PC - Free - Gradio

[**Stable Diffusion XL (SDXL) Locally On Your PC - 8GB VRAM - Easy Tutorial With Automatic Installer**](https://youtu.be/__7VNmnn5iU)

[![image](https://cdn-uploads.huggingface.co/production/uploads/6345bd89fe134dfd7a0dba40/3UAi7i7oLdtvlibSlx4z9.png)](https://youtu.be/__7VNmnn5iU)

**Comment by caniyabanci76** [comment]
This is Joe's repo, not ShivamShrirao's

**Comment by JoePenna** [comment]
@FurkanGozukara please don't spam videos into the discussion here.

Especially ones that aren't related to this repo specifically.

**Reply by FurkanGozukara** [reply]
> @FurkanGozukara please don't spam videos into the discussion here.
> 
> Especially ones that aren't related to this repo specifically.

Sure . Since it was show case I thought related.

**Reply by caniyabanci76** [reply]
@FurkanGozukara, Joe asked you to remove the links, but you reposted them again?

**Reply by FurkanGozukara** [reply]
> @FurkanGozukara, Joe asked you to remove the links, but you reposted them again?

No I didn't touch here last modified date is same

**Reply by caniyabanci76** [reply]
and yet you still won't remove it (it's still at the top) - none of your videos are related to this repo.  this is spamming.
